_data-prepper/pipelines/configuration/processors/date.md

```diff
@@ -40,5 +46,29 @@ The following table describes common [Abstract processor](https://github.com/ope
 
 The `date` processor includes the following custom metrics.
 
-* `dateProcessingMatchSuccessCounter`: Returns the number of records that match with at least one pattern specified by the `match configuration` option. 
-* `dateProcessingMatchFailureCounter`: Returns the number of records that did not match any of the patterns specified by the `patterns match` configuration option.
\ No newline at end of file
+* `dateProcessingMatchSuccessCounter`: Returns the number of records that match with at least one pattern specified by the `match configuration` option.
+* `dateProcessingMatchFailureCounter`: Returns the number of records that did not match any of the patterns specified by the `patterns match` configuration option.
+
+## Example: Add default timestamp to event
+The following Date processor configuration can be used to add default timestamp in `@timestamp` filed to all events:
+
+```yaml
+- date:
+    from_time_received: true
+    destination: "@timestamp"
+```
+
+## Example: Parse timestamp to convert format and time zone
+The following Data processor configuration can be used to parse the value of timestamp filed in `dd/MMM/yyyy:HH:mm:ss` and write it in `yyyy-MM-dd'T'HH:mm:ss.SSSXXX` format:
```

Comment:
"date" instead of "Data"? Please correct capitalization, add articles, and revise for clarity.